packages/features/embed/lib/EmbedTabs.tsx

```diff
@@ -199,32 +199,28 @@ const getEmbedTypeSpecificString = ({
   if (!frameworkCodes[embedType]) {
     throw new Error(`Code not available for framework:${embedFramework} and embedType:${embedType}`);
   }
+
+  const codeGeneratorInput = {
+    calLink,
+    uiInstructionCode: getEmbedUIInstructionString(uiInstructionStringArg),
+    embedCalOrigin,
+    namespace,
+  };
```

Comment:
Now, we have ensured that all methods accept the same structure of data. So, it is possible to use a common object.